GoogleUtilities/ISASwizzler/GULSwizzledObject.m

```diff
@@ -17,7 +17,7 @@
 #import "GoogleUtilities/ISASwizzler/GULObjectSwizzler+Internal.h"
 #import "GoogleUtilities/ISASwizzler/Public/GoogleUtilities/GULSwizzledObject.h"
 
-NSString *kGULSwizzlerAssociatedObjectKey = @"gul_objectSwizzler";
+const NSString *const kGULSwizzlerAssociatedObjectKey = @"gul_objectSwizzler";
```

Comment:
The first `const` is not necessary, right? Since the NSString is immutable and guaranteed to not point to an NSMutableString since we assign it to a string literal. (I'm fine with leaving it as such, as this could be more defensive again future changes)

Reply:
That's true, and since we're using the address of the variable instead of its value in theory it shouldn't matter if the value is mutated as long as its memory address remained the same.